product-service/src/lambda/create-product/validation-scheme.ts

```diff
@@ -0,0 +1,39 @@
+import Ajv from "ajv";
+
+// todo: refactor
+
+const ajv = new Ajv({ allErrors: true, schemaId: "id" });
+ajv.addMetaSchema(require("ajv/lib/refs/json-schema-draft-04.json"));
+
+// TODO: return type coercion
+const createProductScheme = {
+  $id: "http://json-schema.org/draft-04/schema#",
+  $schema: "http://json-schema.org/draft-04/schema#",
+  type: "object",
+  required: ["title", "price", "count", "description"],
+  properties: {
+    title: {
+      type: "string"
+    },
+    description: {
+      type: "string"
+    },
+    price: {
+      type: "string",
+    },
+    count: {
+      type: "string",
+    },

```

Comment:
Shouldn't they be numbers?

Reply:
Yep, sure! came across with ajv type coercion problem, work on a fix

Reply:
fix it, deployed (numbers with boundary limits)